src/workers/workerHelpers.ts

```diff
@@ -322,6 +328,50 @@ function validateEnum(value: string, options: string[] | null): ValidationReturn
     : false;
 }
 
+/**
+ * Calculates the size of a repeated argument.
+ *
+ * @param {string} str - The string representation of the repeated argument.
+ * @returns {number} The size of the repeated argument.
+ */
+function repeatSize(str: string): number {
+  // Remove whitespace and newlines
+  str = str.replace(/\s/g, '');
+
+  // Add quotes to property names
+  str = str.replace(/(\w+)(?=:)/g, '"$1"');
+
+  // add quotes to values
+  str = str.replace(/(?<=:)\b(\w+)\b/g, '"$1"');
+
+  // Remove trailing commas
+  str = str.replace(/,\s*}/g, '}').replace(/,\s*]/g, ']');
+
+  return (JSON.parse(str) as any[]).length;
```

Comment:
This parse seems to be throwing an error if you give the sequence invalid JSON, which I think is causing downstream issues. Here is my sequence I'm testing with:

```ts
export default () =>
  Sequence.new({
    seqId: 'seq0',
    metadata: {},
    steps: [
      A('2020-001T00:00:00').SEQ_VAR_CMD({ opcode: 1, dynamic_args: [{dynamic_arg: ""}] })
    ]
  });
```

Reply:
Should be good to go now. I add back an empty string if it was missing from JSON. 

before:
`[{"dynamic_arg": }]`

now:
`[{"dynamic_arg": ""}]`